web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -407,6 +407,22 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
       previewHost: GesturePreviewHost
     }> = {};
 
+    const clearActiveGestures = (exception?: string) => {
```

Comment:
```suggestion
    const clearActiveGestures = (exceptGestureIdentifier?: string) => {
```

`exception` is confusable with Javascript exceptions...